sdk/communication/Azure.Communication.PhoneNumbers/README.md

```diff
@@ -135,12 +164,66 @@ var releaseOperation = await client.StartReleasePhoneNumberAsync(purchasedPhoneN
 await releaseOperation.WaitForCompletionResponseAsync();
 ```
 
+### SipRoutingClient
+
+#### Retrieve SIP trunks and routes
+
+Get the list of currently configured trunks or routes.
+
+```C# Snippet:RetrieveListAsync
+var trunksResponse = await client.GetTrunksAsync();
+var routesResponse = await client.GetRoutesAsync();
+```
+
+#### Replace SIP trunks and routes
+
+Replace the list of currently configured trunks or routes.
+
+```C# Snippet:ReplaceAsync
+// Cannot delete trunks that are used in any of the routes, therefore first set the routes as empty list, and then update routes.
```

Comment:
Is there a more precise way to do this than setting all routes to an empty list?
```suggestion
// The service will not allow trunks that are used in any of the routes to be deleted, therefore first set the routes as empty list, and then update the routes.
```

Reply:
There is no other way than to set routes to empty list first.
To be honest, there shouldn't be much needed for that, because function: "set_trunks" will probably be called only when first initializing the configuration and later the user will be adding or changing one trunk at a time.